dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/io.openliberty.jakartaee-9.0.feature

```diff
@@ -13,6 +13,8 @@ Subsystem-Name: Jakarta EE Platform 9.0
  com.ibm.websphere.appserver.transaction-2.0,\
  io.openliberty.appAuthorization-2.0,\
  io.openliberty.appClientSupport-2.0,\
+ io.openliberty.batch-2.0,\
+ io.openliberth.batchManagement-2.0,\
```

Comment:
batchManagement was not part of the convenience feature in 8.  Is there a reason for it to be part of it in 9?  liberty is spelled wrong for the feature though.  If you want it to be in the beta image for Jakarta 9 and it doesn't belong in the convenience feature, you should add it to `dev/build.image/profiles/jakartaee9/features.xml`.  Confirm with @tevans78